frontend/src/pages/modelRegistry/screens/ModelRegistrySelectorNavigator.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import ModelRegistrySelector from './ModelRegistrySelector';
+
+type ModelRegistrySelectorNavigatorProps = {
+  getRedirectPath: (namespace: string) => string;
+} & Omit<React.ComponentProps<typeof ModelRegistrySelector>, 'onSelection' | 'modelRegistry'>;
+
+const ModelRegistrySelectorNavigator: React.FC<ModelRegistrySelectorNavigatorProps> = ({
+  getRedirectPath,
+  ...projectSelectorProps
```

Comment:
change this to `modelRegistrySelectorProps`

Reply:
good catch!